classes/submission/Submission.php

```diff
@@ -299,14 +299,7 @@ public function getCoverImageAltText()
     public function getAuthorOrEditorString($preferred = true)
     {
         if ($this->getWorkType() != self::WORK_TYPE_EDITED_VOLUME) {
-            $userGroupIds = array_map(function ($author) {
-                return $author->getData('userGroupId');
-            }, Repo::author()->getSubmissionAuthors($this, true)->toArray());
-
-            $userGroups = array_map(function ($userGroupId) {
-                return DAORegistry::getDAO('UserGroupDAO')->getbyId($userGroupId);
-            }, array_unique($userGroupIds));
-            return $this->getCurrentPublication()->getAuthorString($userGroups);
+            return $this->getAuthorString($preferred);
```

Comment:
`PKPSubmission::getAuthorString` is deprecated.

Reply:
fixed (I think)